.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence
+* @alexcjohnson @Marc-Andre-Rivet @xhlulu @chriddyp @plotly/dash-core
```

```suggestion
* @alexcjohnson @Marc-Andre-Rivet @xhlulu
```

Let's go with the same model dash-bio has - reducing noise for Chris and others in `dash-core`.

I don't think we need to test it, it looks fine - next time someone makes a PR we'll see it in action.